src/content/en/updates/2018/09/chrome-70-media-updates.md

```diff
@@ -0,0 +1,128 @@
+
+{# wf_updated_on: 2018-09-17 #}
+{# wf_published_on: 2018-09-17 #}
+{# wf_tags: news,chrome70,media #}
+{# wf_featured_image: /web/updates/images/generic/play-outline.png #}
+{# wf_featured_snippet: A round up of the audio/video updates in Chrome 70: Cross-codec and cross-bytestream buffering and playback, Opus in MP4 with MSE, and protected content playback allowed by default on Android. #}
+{# wf_blink_components: Blink>Media #}
+
+# Audio/Video Updates in Chrome 70 {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+- [Cross-codec and cross-bytestream buffering and
+  playback](#sourcebuffer-changetype) is possible in MSE.
+- Chrome now supports [Opus in MP4 with MSE](#opus-in-mp4-for-mse).
+- Protected content playback is [allowed by
+  default](#protected-content-allowed-by-default) on Android.
+
+## Support for codec and container switching in MSE {: #sourcebuffer-changetype }
+
+Chrome is [adding support] for improved cross-codec or cross-bytestream
+transitions in [Media Source Extensions] (MSE) playback using a new
+`changeType()` method on `SourceBuffer`. It basically allows the type of media
+bytes subsequently appended to the `SourceBuffer` to be changed.
+
+While the [W3C REC version of MSE] supports adaptive playback of media, that
+adaptation requires that any media appended to a `SourceBuffer` must conform to
+the MIME type provided when initially creating the `SourceBuffer` via
+`MediaSource.addSourceBuffer(type)`, and codecs from that type and any
+previously parsed initialization segments must remain the same throughout. This
+means the website has to take explicit steps to accomplish codec or bytestream
+switching (by using multiple media elements or `SourceBuffer` tracks and
+switching among those), increasing application complexity and user-visible
+latency (such transitions require the web app to take synchronous action on the
+renderer main thread). This transition latency impairs the smoothness of media
+playback across transitions.
+
+With the new `changeType()` method on `SourceBuffer`, a `SourceBuffer` can
+buffer and support playback across different [bytestream formats] and codecs.
+This new method retains previously buffered media modulo future MSE coded frame
+eviction or removal, and leverages the splicing and buffering logic in the
+existing MSE coded frame processing algorithm.
+
+    const sourceBuffer = myMediaSource.addSourceBuffer('video/webm; codecs="opus, vp09.00.10.08"');
+    sourceBuffer.appendBuffer(someWebmOpusVP9Data);
+
+    // Later on...
+    if ('changeType' in sourceBuffer) {
+      // Change source buffer type and append new data.
+      sourceBuffer.changeType('video/mp4; codecs="mp4a.40.5, avc1.4d001e"');
+      sourceBuffer.appendBuffer(someMp4AacAvcData);
+    }
+
+As expected, if the passed type is not supported by the browser, this method
+throws an error.
```

Comment:
Should this be in a `try...catch` block? At least name the error type.

Reply:
I've named the error.
`try...catch` block is featured in the code sample.